skyplane/cli/cli_impl/init.py

```diff
@@ -94,161 +98,98 @@ def make_role_cmds(service_principal, subscription_id):
-                or not config.azure_client_secret
-                or not config.azure_subscription_id
-            ):
-                typer.secho("    Azure credentials not configured correctly, disabling Azure support.", fg="red", err=True)
-                return clear_azure_config(config)
-
-            # authorize new service principal with Storage Blob Data Contributor and Storage Account Contributor roles to the subscription
-            role_cmds = make_role_cmds(config.azure_client_id, config.azure_subscription_id)
-            typer.secho(
-                f"    I will run the following commands to authorize the newly created Skyplane service principal to access your storage accounts:",
-                fg="blue",
-            )
-            for role_cmd in role_cmds:
-                typer.secho(f"        $ {' '.join(role_cmd)}", fg="yellow")
-
-            with Progress(
-                TextColumn("    "),
-                SpinnerColumn(),
-                TextColumn("Authorizing service principal to access storage accounts{task.description}"),
-                transient=True,
-            ) as progress:
-                progress.add_task("", total=None)
-                for role_cmd in role_cmds:
-                    out, err = subprocess.Popen(role_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
-                    if err:
-                        typer.secho(f"    Error running command: {role_cmd}", fg="red", err=True)
-                        typer.secho(f"    stdout: {out.decode('utf-8')}", fg="red", err=True)
-                        typer.secho(f"    stderr: {err.decode('utf-8')}", fg="red", err=True)
-                        return clear_azure_config(config)
-            typer.secho(
-                f"    Azure service principal created successfully! To delete it, run `az ad sp delete --id {config.azure_client_id}`.",
-                fg="green",
-            )
+        typer.secho(
+            f"    Azure managed identity created successfully! To delete it, run `az identity delete -n skyplane_umi -g skyplane`.",
+            fg="green",
+        )
 
         config.azure_enabled = True
         auth = AzureAuthentication(config=config)
-        try:
-            with Progress(
-                TextColumn("    "),
-                SpinnerColumn(),
-                TextColumn("Waiting for Azure client secret to propagate{task.description}"),
-                transient=True,
-            ) as progress:
-                progress.add_task("", total=None)
-                auth.wait_for_valid_token()
-        except TimeoutError:
-            typer.secho(
-                "    The Azure service principal doesn't have access to your storage accounts. Please ensure you have granted the service principal the Storage Blob Data Contributor and Storage Account Contributor roles.",
-                fg="red",
-                err=True,
-            )
-            return clear_azure_config(config)
+        # try:
```

Comment:
Go ahead and remove this code. Also print a log message saying "You may need to wait up to 5 minutes for the Azure managed identity to propagate. If you encounter errors, please wait and retry."